extensions/spatial.sigmf-ext.md

```diff
@@ -0,0 +1,362 @@
+# The `spatial` SigMF Extension Namespace v0.0.2
+
+This document defines the `spatial` extension namespace for the Signal Metadata
+Format (SigMF) specification. This extension namespace contains objects to help
+store information about spatially diverse data, specifically information applied
+to directional antennas and multichannel phase-coherent datasets used for signal
+direction of arrival and beamforming.
+
+Multichannel datasets can be stored in SigMF Collections or as multichannel
+interleaved data depending on the application, Collections are RECOMMENDED due
+to the easier processing and better application support.
```

Comment:
Those might be advantages, but the disadvantages (no intrinsic guarantee of a common Fs, Fc, start_time among channels) are real too.